packages/ui/src/components/va-tree-view/components/VaTreeNode/VaTreeNode.vue

```diff
@@ -97,24 +98,28 @@ export default defineComponent({
     const {
       iconBy,
       selectable,
+      expandNodeBy,
       colorComputed,
+      selectedNodeComputed,
       getText,
       getTrackBy,
       toggleNode,
-      getNodeProperty,
       toggleCheckbox,
+      getNodeProperty,
+      handleKeyboardNavigation,
     } = useStrictInject(TreeViewKey, INJECTION_ERROR_MESSAGE)
 
     const labelComputed = computed(() => getText(props.node) || '')
-    const isExpandedComputed = computed(() => !!props.node.expanded)
+    const isExpandedComputed = computed(() => props.node.hasChildren ? !!props.node.expanded : undefined)
```

Comment:
const isExpandedComputed = computed(() => props.node.hasChildren && !!props.node.expanded)

Reply:
I need here `undefined` for the purpose of removing the `aria-*` attribute if is node expandable/not expandable.